src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -360,7 +361,11 @@ private void sendErrorMessage(
         ErrorCode errorCode
     ) {
         try {
-            sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
+            String message = cause.getMessage();
+            if (cause instanceof OcrValidationException) {
+                message = ((OcrValidationException) cause).getDetailMessage();
+            }
```

Comment:
This is generic code looking for a very specific exception. We need this additional field which won't be logged, so how about adding it to `InvalidEnvelopeException` instead of `OcrValidationException`?

Reply:
ocr exception extends InvalidMetafileException, but we can add. but I did not see any benefit adding there.